modules/openy_features/openy_prgf/modules/openy_prgf_secondary_description_sidebar/config/install/field.field.paragraph.secondary_description_sidebar.field_prgf_right_column_block.yml

```diff
@@ -2,14 +2,11 @@ langcode: en
 status: true
 dependencies:
   config:
-    - block_content.type.basic_block
-    - block_content.type.code_block
-    - block_content.type.date_block
-    - block_content.type.simple_block
     - field.storage.paragraph.field_prgf_right_column_block
     - paragraphs.paragraphs_type.secondary_description_sidebar
   module:
     - datalayer
+    - openy_er
 third_party_settings:
   datalayer:
     expose: 0
```

Comment:
not used in this field, no need for datalayer dependency 3 lines above

Reply:
We do use it there, you can see configs below 
```
    target_bundles_no_dep:
      basic_block: basic_block
      code_block: code_block
      date_block: date_block
      simple_block: simple_block
```